assets/app/views/_labels.html

```diff
@@ -0,0 +1,55 @@
+<div ng-controller="LabelsController" class="labels">
+  <div class="gutter-bottom">
+    <div class="flow">
+      <div class="flow-block">
+        <h2>Labels</h2> 
+      </div>
+      <div class="flow-block right">
+        <ul class="list-inline">
+          <li><a class="action action-inline" href ng-click="toggleExpanded()" ng-show="expanded"><i class="pficon pficon-remove"></i>Collapse</a></li>
+          <li><a class="action action-inline" href ng-click="toggleExpanded()" ng-hide="expanded"><i class="pficon pficon-add"></i>Add labels</a></li>
+          <li><span class="help action-inline"><i class="pficon pficon-help"></i> <a href data-toggle="tooltip" data-placement="bottom" title="Labels are a way to filter objects you create in OpenShift.">About labels</a></span></li>
```

Comment:
@jwforres @liggitt - Do we have an official blurb to describe what labels are? I just made up something here.